lib/tapp.hoon

```diff
@@ -28,6 +30,8 @@
       active=(unit eval-form:eval:tapp-async)
       app-state=state-type
   ==
++$  tapp-peek
+  [%noun ?(? (set contract))]
```

Comment:
My mental parser paused for a second here to verify that `?(?` wasn't some strange irregular syntax it hadn't seen before.